test/functional/TestKernel.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OnMoon\OpenApiServerBundle\Test\Functional;
+
+use OnMoon\OpenApiServerBundle\OpenApiServerBundle;
+use Symfony\Bundle\FrameworkBundle\FrameworkBundle;
+use Symfony\Bundle\FrameworkBundle\Kernel\MicroKernelTrait;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Symfony\Component\HttpKernel\Kernel as BaseKernel;
+use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
+
+class TestKernel extends BaseKernel
```

Comment:
The tests using the kernel generate cache files in /var/cache in the root directory of the bundle
The cache should be :
- generated in the directory of the test
- purged after the tests completes